share/patch_mysql_db_zonemaster_backend_ver_5.0.2.pl

```diff
@@ -0,0 +1,26 @@
+use strict;
+use warnings;
+use utf8;
+use Data::Dumper;
```

Comment:
Remove unused import?

Reply:
I just copied the "5.0.0" script and changed as little as possible. Wouldn't it be better to do that for next release in all five scripts?

Reply:
Ah, I didn't look that far. Sure!